packages/frontend/src/components/Routing.js

```diff
@@ -199,7 +200,6 @@ class Routing extends Component {
         } = this.props;
 
         fetchTokenFiatValues();
-        this.startPollingTokenFiatValue();
```

Comment:
Are we no longer polling the `/fiat` endpoint? Should this be feature flagged?

Reply:
After talking with @Patrick1904 and @MaximusHaximus we've decided to revert this change and keep fiat polling in `Routing`